tests/ui/parser/parser-ice-ed2021-await-105210.rs

```diff
@@ -0,0 +1,10 @@
+// ICE #105210 self.lines.iter().all(|r| !r.iter().any(|sc| sc.chr == \'\\t\'))
+// ignore-tidy-tab
```

Comment:
Is this necessary?

Reply:
I like to have some context in the test what the ice looked like

Reply:
No, I mean `ignore-tidy-tab`.

Reply:
Ah lol.
The original repro needed a tab on a very specific place for some reason :octopus: 
while reducing the ice would go away if I replaced the tab with a space